sirepo/feature_config.py

```diff
@@ -116,6 +106,10 @@ def b(msg, dev=False):
         # No secrets should be stored here (see sirepo.job.agent_env)
         api_modules=((), set, 'optional api modules, e.g. status'),
         default_proprietary_sim_types=(set(), set, 'codes where all users are authorized by default but that authorization can be revoked'),
+        gui=dict(
```

Comment:
slack_uri does not belong in gui, because it is only used server-side (like api_modules, etc.). 

Now that I see what this looks like, I would call this schema_common just to be clear that this gets added to the schema.

You need to ReplacedBy sim_common.hide_guest_warning look in pkconfig for how to do that.